notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -400,6 +400,46 @@ alcohol abuse [Ruby and her mother discussed her father's alcoholism.]
 tax evasion [It turned out that Maggie Ann Ford had traveled back in time to avoid being prosecuted for this crime.]
 
 
+movie: Possible Worlds (2000)
+=============================
+
+:: Title
+Possible Worlds
+
+:: Description
+Possible Worlds is a 2000 Canadian film adaptation of the 1990 play of the
+same name by John Mighton.
+
+Synopsis: A mathematician falls in love with the same woman in multiple
+parallel realities, as a police detective tries to apprehend a brain stealing
+serial killer.
+
+:: Date
+2000-09-04
+
+:: References
+https://en.wikipedia.org/wiki/Possible_Worlds_(film)
+
+:: Ratings
+2 <paul>
+
+:: Choice Themes
+parallel universe [The film explores the experiences of the mathematician George Barber as he falls in love with a woman named Joyce in multiple parallel realities. Hence the title "Possible Worlds".]
+
+:: Major Themes
+law enforcement [One storyline follows Inspector Berkley as he attempts to apprehend a serial killer who was stealing peoples brains.]
+serial murder [A brain stealing serial killer was on the loose. It turned out to be a neuroscientist who was stealing brains for his experiments on consciousness.]
+romantic love [George fell in love with Joyce in multiple parallel realities. In one reality she was a scientist, in another she was a stockbroker, and in another she didn't seem to remember him from on moment to another. But he loved each version of her nonetheless.]
+brain disembodiment technology [George's disembodied brain was hooked up to a life-support system from a certain point in the story. Inspector Berkley was fascinated with the brain of a rat that was kept alive in a jar, and he took it back to his office for further observation.]
```

Comment:
Conceivably, we might need a "speculative brain disembodiment effect" one day..!

Reply:
Hah, yes, an all-purpose "this is a story about a disembodied brain" kind of theme.